.github/actions/setup-spack-stack/action.yaml

```diff
@@ -84,10 +84,14 @@ runs:
         #brew install curl
         #brew install git
         #brew install git-lfs
+        # For now we need gcc-10/gfortran-10
+        brew install gcc@10
         brew install lmod
         brew install qt@5
         brew install readline
         brew install wget
+
+        pkgutil --pkg-info=com.apple.pkg.CLTools_Executables
```

Comment:
This is useful to have, it writes the Xcode command line utilities version to the log file